app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -129,13 +134,15 @@ class PendingTransactionTracker extends EventEmitter {
   }
 
   /**
-    Ask the network for the transaction to see if it has been include in a block
-    @param {Object} txMeta - the txMeta object
-    @emits tx:failed
-    @emits tx:dropped
-    @emits tx:confirmed
-    @emits tx:warning
-  */
+   * Query the network to see if the given {@code txMeta} has been included in a block
+   * @param {Object} txMeta - the transaction metadata
+   * @returns {Promise<void>}
```

Comment:
Technically it does return `Promise<bool>` in some cases (where `this.emit` is returned)

Reply:
I was going to update that separately—that return shouldn't have that value